composables/httpd.ts

```diff
@@ -0,0 +1,22 @@
+const PER_PAGE = 100
+
+function createHttpdBaseURL(domainName = 'seed.radicle.xyz') {
+  return `https://${domainName}/api/v1`
+}
+
+interface FetchOptions {
+  domainName?: string
+}
+
+interface UseFetchIssuesOptions extends FetchOptions {
+  rid: string
+}
+
+export async function useFetchIssues({ rid, domainName }: UseFetchIssuesOptions) {
+  return await useHttpdFetch('/projects/{rid}/issues', {
+    path: { rid },
+    baseURL: createHttpdBaseURL(domainName),
```

Comment:
`baseUrl` should be set in the plugin. Otherwise, we'll have to create a different fetch function for each endpoint (useFetchIssues, useFetchProjects, ...) which is half of what we were trying to avoid by leveraging the OpenAPI schema.

For the scope of this issue it's ok to hardcode the baseUrl, but keep in mind that default should eventually be _exactly_ (scheme, domain, port, path) whatever httpd is listening at by default.

The above is scoped for Issue #8 but if you'd prefer to do that in this PR that's fine by me. One could argue it should have been in this one in the first place. Up to you.

Reply:
Got it, wasn't sure what was available for us in the plugins (guess I need to do some reading up on that)

I have a feeling I'll need it for #18, so I'll revisit this there.